README.md

```diff
@@ -7,9 +7,11 @@
 - [Octicons](https://www.figma.com/file/1ljgTFkT5NKNRfq5hw07JQ/Octicons?node-id=0%3A1&viewport=664%2C488%2C1)
 
 ## 📖 Documentation
-
 Information on how to use Primer within Figma lives in our [design guidelines](https://primer.style/design/tools/figma). You'll be able to find detailed documentation on how GitHub uses Figma.
 
 ## 🌐 Community
 
 Primer's component libraries, shared styles, and Octicons can be found in the Figma community at https://www.figma.com/@primer.
+
+## Help & questions
```

Comment:
This is an external public facing repo, we cannot add slack messages as a link

Reply:
But as a text is fine?